src/assets/flags.js

```diff
@@ -0,0 +1,11 @@
+import FlagPl from '../assets/flags/flag_pl.png';
+import FlagGbr from '../assets/flags/flag_gbr.jpg';
+import FlagDeu from '../assets/flags/flag_deu.png';
+import FlagFr from '../assets/flags/flag_fr.jpeg';
+
+export const flags = {
+  flagPl: FlagPl,
+  flagGbr: FlagGbr,
+  flagDeu: FlagDeu,
+  flagFr: FlagFr,
```

Comment:
jeśli klucz i wartość maja taką samą nozwę to nie musisz pisać nazwy wartości. np: 
```
export const flags = {
  flagPl,
  flagGbr,
}
```